packages/integration/lib/src/widgets/lists/feed/feed_list.dart

```diff
@@ -134,7 +134,8 @@ class _LMFeedListState extends State<LMFeedList> {
   Widget build(BuildContext context) {
     return BlocListener<LMFeedUniversalBloc, LMFeedUniversalState>(
       bloc: _feedBloc,
-      listener: (context, LMFeedUniversalState state) => updatePagingControllers(state),
+      listener: (context, LMFeedUniversalState state) =>
```

Comment:
here we can reference the `updatePagingControllers` function directly instead of calling it.

Reply:
Okay

Reply:
then we can update `updatePagingControllers()` method to take `BuildContext` as an additional args

Reply:
Okay